docs/react-native/docs/ios/remote-notification-support.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Remote Notification support
+description:  Utilise the power of Notifee with remote notifications.
+next: /react-native/docs/ios/remote-notification-support
+previous: /react-native/docs/ios/remote-notification-support
+---
+
+## Remote Notification Support
+
+It's possible to display a notification with Notifee features from outside the app using remote notifications (a.k.a push notifications).
+
+By adding a custom key `notifee_options` in the message payload, the notification will be modified by Notifee before it is finally displayed to the end user.
+
+To get started, you will need to implement a [Notification Service Extension](https://developer.apple.com/documentation/usernotifications/unnotificationserviceextension) for your iOS app, the following steps will guide you through how to set it up.
+
+## Add the notification service extension
+
+* From Xcode top menu go to: File > New > Target...
+* A modal will present a list of possible targets, scroll down or use the filter to select Notification Service Extension. Press Next.
+* Add a product name (use `NotifeeNotificationService` to follow along) and click Finish
+
+<Vimeo id="remote-notification-support-1" caption="Step 1 - Add Your Notification Service Extension" />
+
+## Add target to the Podfile
+
+Ensure that your new extension has access to `NotifeeExtensionHelper` by adding it in the Podfile:
+* From the Navigator open the Podfile: Pods > Podfile
+* Scroll down to the bottom of the file and add
+
+```yaml
+target 'NotifeeNotificationService' do
+  pod 'RNNotifee/NotifeeExtensionHelper'
+end
+```
+
+* Install or update your pods using pod install from the ios folder
+
+`pod install --repo-update`
+
+<Vimeo id="remote-notification-support-2" caption="Step 2 - Update Your Pods" />
+
+## Use the extension helper
+
+At this point everything should still be running normally. This is the final step which is invoking the extension helper.
+
+* From the navigator select your NotifeeNotificationService extension
+* Open the NotificationService.m file
+* At the top of the file import NotifeeExtensionHelper.h right after the NotificationService.h as shown below
+```
```

Comment:
objective-c I think does markup right (I seem to remember somewhere obj-c wasn't as compatible)

Reply:
used this as a reference https://github.com/github/linguist/blob/master/lib/linguist/languages.yml, went with objectivec